superset/models/slice.py

```diff
@@ -206,7 +206,7 @@ def digest(self) -> str:
         """
         Returns a MD5 HEX digest that makes this dashboard unique
         """
-        return utils.md5_hex(self.params)
+        return utils.md5_hex(self.params or "")
```

Comment:
Not that it matters, but I've always been allergic to md5 on empty string:
```
>>> import hashlib
>>> hashlib.md5(''.encode()).hexdigest()
'd41d8cd98f00b204e9800998ecf8427e'
```

```suggestion
        return utils.md5_hex(self.params) if self.params else ''
```

Reply:
I think I'll return `None` here just to be more explicit. The original change is for avoiding an uncaught exception, explicitly returning None forces the downstream handle the case of missing `params` more properly.

Reply:
Nvm. I'll keep `utils.md5_hex(self.params or "")` since this key is also used in thumbnail URLs:

```python
        return f"/api/v1/chart/{self.id}/thumbnail/{self.digest}/"
```

We can make a follow up PR to rename `digest` to `thumbnail_key` (for both chart and dashboard) if we think it's necessary.